src/Microsoft.SqlTools.ServiceLayer/DacFx/DacFxService.cs

```diff
@@ -329,7 +330,7 @@ public async Task HandleSavePublishProfileRequest(SavePublishProfileParams param
                     DacProfile profile = new DacProfile();
                     profile.TargetDatabaseName = parameters.DatabaseName;
                     profile.TargetConnectionString = parameters.ConnectionString;
-                    //TODO: Set deploy options to pass on to DacFx
+                    profile.DeployOptions = SchemaCompareUtils.CreateSchemaCompareOptions(parameters.DeploymentOptions);
```

Comment:
I had the same thought while working on the changes, because the name and location of the method should be generic instead of being specific to Schema Compare. Updated it appropriately.